src/showcase/app/howtoupdate/how-to-update.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+
+// @ts-ignore versions.ts is generated automatically by bazel
+import { libraryVersion } from '../versions';
+
+@Component({
+  selector: 'sbb-how-to-update',
+  templateUrl: './how-to-update.component.html',
+})
+export class HowToUpdateComponent {
+  libraryMajorVersion: number = +libraryVersion.split('.')[0];
+  libraryPredecessorMajorVersion = this.libraryMajorVersion - 1;
+  form: FormGroup = new FormGroup({ fullCommand: new FormControl(false) });
```

Comment:
I would prefer to just have a simple FormControl. There is no need for a FormGroup here.